### DISCUSSION TITLE
Menus vs. Toolbar Icons

### DISCUSSION THREAD

**Discussion by GeoRW** [GeoRW/ACBF-Editor#22] [General]
I see that GTK4 version has a menu icon with items in it instead of toolbar icons which are used in GTK3. I would rather prefer toolbar icons as it is much simpler and easier to use interface. For example, to go into "Frames Editor", you need to click on menu icon, then find that menu item there and then click again, while with toolbar icons you need just one click. Was this done because GTK4 does not have a specific toolbar class now?

**Comment by mizaki** [comment]
It was done because of the general GTK4 style of things now. And, I'd need to recreate it.

Taking your example, you have a button that is only clicked once taking up usable space. Where you're interacting with things over and over again, having a toolbar makes more sense (like the one that is mostly empty atm in Frames/Texts). I've added a shortcut key to the main menu items now which I think makes the toolbar icons even less required?

It could also be added as an option.

**Reply by mizaki** [reply]
Could also add the frames/text icon next to `open`, `save`. I don't think any of the others would get enough use to warrant it?

**Reply by GeoRW** [reply]
I personally hate fishing stuff from menus. When I'm creating a new book to add to my library, I'm using all of them besides "Preferences" and "About" button. So, if you remove these 2 and add menu button, you will just save space for one icon.
And it just looks ugly without icons.

**Reply by mizaki** [reply]
It's hardly a large menu :) Shortcut keys are even quicker!

Are you really going into "Fonts/Styles" and "Languages" all that often? Even "Content" can't be that much?

I think I'll move the menu button over to the right (and language?). Then, how about: Open, Save, spacer, Frames/Texts (and there should still be enough space for fonts, contents)?

**Reply by GeoRW** [reply]
I didn't say it's large menu, what I'm saying is that user needs to go into the menu and fish it from there. And why would I learn shortcut keys? Shortcut keys are good to learn if they are consistent across applications like CTRL+B for bold text, CTRL+U for underline etc.
When I'm creating a new book I testing a lot of different fonts, so yes I'm going there quite often.
Also contents I use quite often as I never add single issues to my library for US comics, I always add collected editions/TPB ... and EU comics often have chapters.
By language you mean what? Languages dropdown? That one is convenient to use when doing translation to switch between one and the other.

**Reply by GeoRW** [reply]
Regarding Fonts/Styles that can be moved to Font Editor window as it is more convenient to use there.
